components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/UserEmailVerificationHandler.java

```diff
@@ -420,7 +420,13 @@ private void triggerNotification(User user, String type, String code, Property[]
 
         String eventName = IdentityEventConstants.Event.TRIGGER_NOTIFICATION;
 
+              String serviceProviderUUID = (String) IdentityUtil.threadLocalProperties.get()
```

Comment:
Check the indentation and let's get rid of the new line

Reply:
ack